crowd_anki/representation/json_serializable.py

```diff
@@ -6,8 +6,8 @@
 
 class JsonSerializable:
     readable_names = {}
-    export_filter_set = {"mod",  # Modification time
-                         "usn",  # Todo clarify
+    export_filter_set = {  # "mod",  # Modification time
+                           # "usn",  # Todo clarify
```

Comment:
hrm, previously Anki would set those to defaults if they were missing. I supposed we can just steal code from previous version =\

Reply:
actually this didn't break for me for some reason?

Reply:
Must've been my fault then.